gravitee-apim-portal-webui-next/src/scss/theme/theme.scss

```diff
@@ -0,0 +1,43 @@
+@use '@angular/material' as mat;
```

Comment:
This should have inserted a license. Does `lint:fix` do that on your local computer?